buildspecs/j9.flags

```diff
@@ -165,6 +165,10 @@
 		<description>Buildspec compiles sources with openjdk.</description>
 		<ifRemoved>Openj9 clone,make jobs longer run on this buildspec.</ifRemoved>
 	</flag>
+	<flag id="build_openj9JDK8">
+		<description>Buildspec compiles sources with openjdk-jdk8.</description>
+		<ifRemoved>Openj9 compile job, sanity and extended tests longer run on this buildspec.</ifRemoved>
```

Comment:
This should say "no longer" instead of "longer". OpenJ9 should have an upper case j.

Reply:
Fixed, thx.